fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/ATSU/A320_Neo_CDU_AOC_Init.js

```diff
@@ -152,8 +152,12 @@ class CDUAocInit {
         if (mcdu.aocTimes.in) {
             inTime = `${FMCMainDisplay.secondsTohhmm(mcdu.aocTimes.in)}[color]green`;
         }
-        if (mcdu.simbrief["blockTime"]) {
-            blockTime = `${FMCMainDisplay.secondsTohhmm(mcdu.simbrief.blockTime)}[color]green`;
+        // Not using simbrief block time anymore
+        // if (mcdu.simbrief["blockTime"]) {
+        //     blockTime = `${FMCMainDisplay.secondsTohhmm(mcdu.simbrief.blockTime)}[color]green`;
+        // }
+        if (mcdu.aocTimes.in && mcdu.aocTimes.out) {
```

Comment:
Please don’t comment out code and leave it. We can easily see the old code in git history if required. Do make sure the PR title accurately reflects what’s in the PR to help people looking at the history in future though.

Reply:
Removed the old code, thanks for letting me know!